### PR TITLE
Jpa implements

### PR DESCRIPTION
**Pull request by KDanil2000**

### CODE REVIEW

File: src/main/java/model/Book.java [hunk 1]
```diff
@@ -26,4 +30,5 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    private boolean isDeleted;
```
Comment (NazarSmith228) [inline src/main/java/model/Book.java:33]: shouldn't you specify the column name like:
```suggestion
    @Column(name = "is_deleted")
    private boolean isDeleted;
```

or it works as it is?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,8 +1,8 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/bookstore
-spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
-spring.datasource.username=root
-spring.datasource.password=Donetsk228322!
-spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
+    spring.datasource.url=jdbc:mysql://localhost:3306/bookstore
+    spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+    spring.datasource.username=root
+    spring.datasource.password=Donetsk228322!
+    spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
```
Comment (NazarSmith228) [inline src/main/resources/application.properties:5]: what are these changes about? some formatting stuff?

**Review by NazarSmith228 — APPROVED** [review]
LGTM! But please pay attention to my comments

File: src/main/resources/application.properties (hunk 2 above)
Comment (KDanil2000) [reply src/main/resources/application.properties:5]: Yes I don't know why it's did like this) I'll fix it)

File: src/main/java/model/Book.java (hunk 1 above)
Comment (KDanil2000) [reply src/main/java/model/Book.java:33]: I think i should to do this)

**Pull request merged**